sympy/stats/tests/test_continuous_rv.py

```diff
@@ -318,6 +318,13 @@ def test_gamma_inverse():
     X = GammaInverse("x", a, b)
     assert density(X)(x) == x**(-a - 1)*b**a*exp(-b/x)/gamma(a)
 
+def test_gompertz():
+    b = Symbol("b", positive=True)
+    eta = Symbol("eta", positive=True)
+
+    X = Gompertz("x", b, eta)
+    assert density(X)(x) == b*eta*exp(b)*exp(-b*exp(eta*x))*exp(eta*x)
```

Comment:
This might be the why: 

```python```
    _argnames = ('eta', 'b')
```

And:

```python

    def check(eta, b):
        _value_check(eta > 0, "eta must be positive")
        _value_check(b > 0, "b must be positive")
```

The variables are clearly interchanged here.